frontend/src/views/FuelCodes/AddFuelCode/_schema.jsx

```diff
@@ -186,22 +145,28 @@ export const fuelCodeColDefs = (t, optionsData) => [
     field: 'fuelCode',
     headerName: t('fuelCode:fuelCodeColLabels.fuelCode'),
     cellDataType: 'text',
+    cellRenderer: (params) =>
+      params.value ||
+      (!params.value && <Typography variant="body4">Select</Typography>),
     cellEditor: 'autocompleteEditor',
-    editable: true,
     cellEditorParams: {
-      onDynamicUpdate: (fuelCodeValue, params) =>
-        autoFill(
-          { fuelCodeValue, latestFuelCodes: optionsData.latestFuelCodes },
-          params
-        ),
-      autoSelect: true,
+      onDynamicUpdate: (params) => params.api.redrawRows({ rowNodes: [params.node] }),
```

Comment:
autoFill method was updating only the cells that aren't visible within the grid view, hence had to use redraw data row and using onRowEditingStopped handler to update the row.